Gemfile

```diff
@@ -7,3 +7,5 @@ gem 'scss_lint', require: false# require flag is necessary https://github.com/br
 gem "reek"
 gem "puppet-lint"
 gem "csvlint"
+gem "img_checker"
+gem "fastimage"
```

Comment:
not sure, but I was having problem in running it, so I added it, and then it started working